data/json/recipes/recipe_electronics.json

```diff
@@ -1983,5 +1983,21 @@
     [ [ "amplifier", 1 ] ],
     [ [ "cable", 2 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "electrolysis_kit",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "using": [ [ "soldering_standard", 10 ] ],
+  "difficulty": 2,
+  "time": 10000,
+  "reversible": true,
+  "autolearn": true,
+  "qualities" : [ {"id":"CUT","level":1} ],
+  "components": [
+    [ [ "cable", 50 ] ],
+    [ [ "scrap", 2 ] ]
+  ]
```

Comment:
Either the electrolysis kit or the tinplating recipe needs some kind of insulated container (glass).  As-is there's nothing to put the acid bath in.